e2e/tests/signup/successfulSignUp.test.js

```diff
@@ -18,7 +20,13 @@ describe(`Successful Sign Up Scenarios`, () => {
     let homePage = null;
 
     beforeAll(async() => {
-        browser = await chromium.launch({headless:false});
+        browser = await chromium.launch({
+            headless:false,
+            logger: {
```

Comment:
how did you verify that the logger works as expected?

Reply:
I check the messages that are printed on my terminal during the execution.

Reply:
OK, also in order to verify that a logger works as expected, you could make a test case fail **on purpose**, in order to verify that it will catch the error.

For example you could change the expected text message on an assertion for the test to fail, and see what the logger returns.

Reply:
Ok! Could you please check my new commit?

Reply:
you don't need to verify the length of the logs. It was just a proposal from my side, a "way" let's say to verify that your code was working as expected.

In general, when you are implementing something, especially for assertions, while "debugging" a good way to check you code, is to actually make it "fail", and when you are sure that it will work as expected, then you can change it in order to work.

It's like you are testing your code.

Reply:
Ok! I got it!